networkx/algorithms/planarity.py

```diff
@@ -1400,3 +1401,63 @@ def copy(self, as_view=False):
             for v, datadict in nbrs.items()
         )
         return G
+
+    def to_undirected(self, reciprocal=False, as_view=False):
+        """
+        Returns an undirected representation of the planar embedding for directed graphs.
```

Comment:
My understanding is that the PlanarEmbedding class only works for undirected graphs. Even though it is a subclass of `DiGraph`, it is representing undirected graphs. So I think this wording should change to something like "non-embedded undirected graph":
```suggestion
        Returns a non-embedding undirected representation of the graph.
```